#
# In the "solaris" sandbox at the pre-authentication phase, many privileges are
# deleted from the privilege separation child process. This patch is to remove
# two additional privileges from the solaris sandbox, PRIV_DAX_ACCESS and
# PRIV_SYS_IB_INFO. 
# 
# We have contributed back this patch to the OpenSSH upstream community (see
# https://bugzilla.mindrot.org/show_bug.cgi?id=2723). In the future, if this
# patch is accepted by the upsteam in a later release, we will remove this
# patch when we upgrade to that release.
#
--- orig/sandbox-solaris.c	2017-05-31 16:54:46.312419035 -0700
+++ new/sandbox-solaris.c	2017-05-31 16:54:46.270304505 -0700
@@ -62,6 +62,12 @@
 #ifdef PRIV_NET_ACCESS
 	    priv_delset(box->pset, PRIV_NET_ACCESS) != 0 ||
 #endif
+#ifdef PRIV_DAX_ACCESS
+	    priv_delset(box->pset, PRIV_DAX_ACCESS) != 0 ||
+#endif
+#ifdef PRIV_SYS_IB_INFO
+	    priv_delset(box->pset, PRIV_SYS_IB_INFO) != 0 ||
+#endif
 	    priv_delset(box->pset, PRIV_PROC_EXEC) != 0 ||
 	    priv_delset(box->pset, PRIV_PROC_FORK) != 0 ||
 	    priv_delset(box->pset, PRIV_PROC_INFO) != 0 ||
